src/Helpers/Validations/ValidationRegex.js

```diff
@@ -0,0 +1,23 @@
+export const validateRegex ={
+    alphaOnly: '^[a-zA-Z .]*$', // alphabet space and dot
+    //alphaOnly: '^[a-zA-Z ]*$',// alphabet and space
+    //alphaOnly: '/^[A-Za-z\\s]+$/',
+    // passwordPattern:
+    //   '/^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[$@$!%*?&#])[A-Za-z\\d$@$!%*?&#]{8,20}/',
+  
+    // passwordPattern:
+    //   '/^(?=\\S*[a-z])(?=\\S*[A-Z])(?=\\S*\\d)(?=\\S*[^\\w\\s])\\S{8,}$/',
```

Comment:
Suggestions: remove commented code from files

Reply:
will remove it after complete development